### DISCUSSION TITLE
The dimensionless temperature in Case 05

### DISCUSSION THREAD

**Discussion by ZZZYunpeng** [GeoChemFoam/GeoChemFoam#45] [Q&A] [answered]
Dear professors,

Hi, I hope you are doing well.

I have a question that what the definition of dimensionless temperature is in Case 05.

In your paper, you defined the dimensionless T*, which is equal to T/(T1-T0).  
![1710938016897](https://github.com/GeoChemFoam/GeoChemFoam/assets/140990976/219b795d-5c99-41d4-a318-f920c892edc0)

![image](https://github.com/GeoChemFoam/GeoChemFoam/assets/140990976/52929cc8-1775-4ac5-a0ed-7ed0c77dd9f7)

T0 is the inlet temperature, T1 is the outlet and walls temperature. What is the " T " in the formula?  Maybe it should be ΔT? 
And should not the temperature of injection fluid in the inlet is equal to the inlet temperature? Why do you choose the property of CO2 at 60 ℃ instead of 50 ℃？

Best regards,
Yunpeng Zhang

**Comment by julienmaes** [comment, marked as answer]
Indeed, T* should be (T-T0)/(T1-T0)

The properties are chosen at 60 degree because it is the median between the two temperatues T0 and T1
